scripts/run_simulator.sh

```diff
@@ -11,34 +11,31 @@ if ! [[ "$0" =~ scripts/run_simulator.sh ]]; then
   exit 255
 fi
 
-# Load the versions
 SUBNET_EVM_PATH=$(
   cd "$(dirname "${BASH_SOURCE[0]}")"
   cd .. && pwd
 )
-source "$SUBNET_EVM_PATH"/scripts/versions.sh
-
 # Load the constants
 source "$SUBNET_EVM_PATH"/scripts/constants.sh
 
 run_simulator() {
-    #################################
-    echo "building simulator"
-    pushd ./cmd/simulator
-    go build -o ./simulator main/*.go
-    echo
-
-    popd
-    echo "running simulator from $PWD"
-    ./cmd/simulator/simulator \
-        --endpoints="$RPC_ENDPOINTS" \
-        --key-dir=./cmd/simulator/.simulator/keys \
-        --timeout=300s \
-        --workers=1 \
-        --txs-per-worker=50000 \
-        --batch-size=50000 \
-        --max-fee-cap=1000000 \
-        --max-tip-cap=10000
+  #################################
+  echo "building simulator"
+  pushd ./cmd/simulator
+  go build -o ./simulator main/*.go
+  echo
+
+  popd
+  echo "running simulator from $PWD"
+  ./cmd/simulator/simulator \
+    --endpoints="$RPC_ENDPOINTS" \
+    --key-dir=./cmd/simulator/.simulator/keys \
+    --timeout=300s \
+    --workers=1 \
+    --txs-per-worker=50000 \
+    --batch-size=50000 \
+    --max-fee-cap=1000000 \
+    --max-tip-cap=10000
```

Comment:
nit ideally revert this, these are auto-formatting out of scope changes

Reply:
I think this actually fixes the formatting (we used 2 spaces) but reverted it, but I agree we should probably find a linter plugin for these.